Automattic/woocommerce-payments
# Add a warning when there are extensions incompatible with WooPay

#### Changes proposed in this Pull Request
This PR introduces e warning for merchants when incompatible plugins are installed. This will not prevent merchants from activating WooPay. The list of incompatible extensions is updated once a day.

Fixes #6422

#### Testing instructions
- Pull the latest changes from WooPay
- Checkout to this branch in your merchant site.
- Go to the WooPay \wp-admin > WooPay and add a plugin that you have installed in in your merchant site to the list of incompatible plugins (It's just the plugin name like woocommerce-payments not woocommerce-payments/woocommerce-payments.php )
- Save
- Go to your merchant site
- Install a plugin that allows you to manage cron jobs manually
- Execute the `validate_incompatible_extensions` cron
- Go to \wp-admin > Payments > Settings > Express checkout
- Make sure the message below shows up

<img width="701" alt="Screenshot 2023-06-09 at 11 03 25 AM" src="https://github.com/Automattic/woocommerce-payments/assets/6342964/b65ad610-d35c-4bfe-b400-8db4a125da4d">

- Remove the extension from the incompatible ones and run the cron again. The warning should not show up anymore

-------------------


**Post merge**
